packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -48,9 +65,9 @@ export default  class FileInput extends UIPlugin {
     }
   }
 
-  handleInputChange (event) {
+  private handleInputChange(event: Event) {
     this.uppy.log('[FileInput] Something selected through input...')
-    const files = toArray(event.target.files)
+    const files = toArray((event.target as HTMLInputElement).files ?? [])
```

Comment:
```suggestion
    const files = toArray((event.target as HTMLInputElement).files!)
```
Alternatively, we could create a `HTMLFileInputElement` to tell TS that we know it has `type="file"` attribute and we don't expect it to change